app/assets/stylesheets/portrait/_base.scss

```diff
@@ -34,6 +34,14 @@
 		}
 		.dropdown-menu {
 			background-color: $color_topnav_background;
+			li {
+				a {
+					&:hover {
+						background: $color_black;
```

Comment:
I'm looking at the base palette: Should the colors here be `$color_black` and `$color_white` explicitly, or should they be related to `$color_topnav: #eee;` and `$color_topnav_background: #222;`? I guess the question is whether we want to vary this by palette or not.

Reply:
like what if we used:
```css
&:hover: {
  background-color: brightness(86%);
  color: brightness(114%);
} 
}
```

Reply:
or, alternately, just document that it's always very dark gray -> black

Reply:
to get the contrast filter to work in a fiddle I had to inherit `color` and `background-color`

Reply:
i think i've found a solution for this case in latest wip